content/en/docs/integration/create-your-own-pair.md

```diff
@@ -49,7 +49,13 @@ Then, you may execute the contract with the organized JSON above.
 
 ## Provide initial liquidity
 
-**Dezswap** pair contract knows the swap rate by the both of the remained assets on the pool. But if you have just created your own pair but no liquidity provided, The contract cannot calculate the rate and all swap & swap simulation raise fail. So, **Dezswap** UI does not list the pair unless the initial liquidity is provided. So, if you want finalize the listing, you should provide the initial liquidity and it should be done on CLI.
+**Dezswap** pair contract derives the swap rate from the amount of the remained assets on the pool. However, if you have just created your own pair and haven't provided its liquidity yet, the contract won't be able to calculate the rate and all swap simulations and transactions will fail. So, for the pair to work successfully, you should provide the initial liquidity.
+
```

Comment:
It would be nice to add the contents of the increase_allowance to the factory address.

Reply:
[9ea8990](https://github.com/dezswap/docs/commit/9ea8990338e7654a7e7866831cf43c75542a4e26)